src/test/java/io/opentelemetry/OverheadWithExternalsTests.java

```
@@ -111,34 +117,36 @@ private void runAppOnce(TestConfig config, Agent agent) throws Exception {
   private void runApp(TestConfig config, Agent agent) throws Exception {
     verifyExternals();
 
-    GenericContainer<?> petclinic = new PetClinicRestContainer(NETWORK, agent, namingConventions, getPostgresHost(), getCollectorHost()).build();
-    long start = System.currentTimeMillis();
-    try {
+    try (GenericContainer<?> petclinic =
+             new PetClinicRestContainer(NETWORK, agent, namingConventions, getPostgresHost(), getCollectorHost())
+                 .build()) {
+      long start = System.currentTimeMillis();
+
       logger.info("Starting petclinic container");
       petclinic.start();
-    } finally {
-      logger.info("Petclinic container has started or failed to start.");
```

Bummer to miss this because the intent was to give insight on the timing when the container startup fails. Does it fail fast or does it take a while? I dunno.

---

Let's add it back then